### PR TITLE
Clarify tracestate header size

### PR DESCRIPTION
**Pull request by mwear**
This addresses issue #310.

Previously, the spec listed tracestate limits in two places, and used the phrasing at most 512 bytes, in one sentence, and at least 512 characters in another. The intent that is that tracing systems should propagate 512 characters, but can choose to propagate more or less depending on their situation. This also makes the change that white space characters  are included in the 512 character recommendation.

The two sentences have been consolidated and clarified under the tracestate limits section.

If there are any suggestions on wording, I'd be happy to address them.

### CODE REVIEW

**Review by discostu105 — APPROVED** [review]

**Review by danielkhan — APPROVED** [review]

File: spec/20-http_header_format.md
Comment (yurishkuro) [inline spec/20-http_header_format.md]: we don't use italic for MUST/SHOULD words, capitalization is enough.

**Review by yurishkuro — COMMENTED** [review]
This PR is a small semantic change to the spec. Do we keep a  changelog?

**Review by mtwo — APPROVED** [review]
Approved pending resolution of Yuri's comment

**Review by AloisReitbauer — APPROVED** [review]

File: spec/20-http_header_format.md
Comment (mwear) [reply spec/20-http_header_format.md]: We decided that we will not be using italics going foward, see [this comment](https://github.com/w3c/trace-context/pull/307#issuecomment-511920186), but will address removal in a future PR.

**Comment by mwear** [comment]
This has been updated so that it's no longer conflicted after merging #307.

**Pull request merged**
